api/src/controllers/middleware.ts

```diff
@@ -40,6 +41,8 @@ function errorHandler(): ErrMiddleware {
         if (err instanceof HttpException) {
             // Handle a known server error
             outErr = err;
+        } else if (err.name === 'UnauthorizedError') {
```

Comment:
Nit: Instead of relying on this name, we can actually use `err instance of UnauthorizedError` from `express-jwt`

```ts
import { UnauthorizedError } from 'express-jwt';
```

and I think we can update the tests as well

Reply:
Omg yes this is WAY BETTER! Thanks Armi, great catch!